src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -253,7 +248,8 @@ object DepositHandler {
 
             if (missingFilesNotInFetchText.isEmpty)
               noFetchItemsAlreadyInBag(bagDir, itemsFromBagStore)
-                .flatMap(_ => validateChecksumsFetchItems(bag, itemsFromBagStore))
+                .flatMap(_ => bagStoreSettings.map(implicit bs => validateChecksumsFetchItems(bag, itemsFromBagStore))
+                  .getOrElse(Failure(new NoSuchElementException("BagStore is not configured - SOMETHING WENT WRONG, YOU SHOULD NEVER REACH THIS PART OF CODE!"))))
```

Comment:
Because:
    val (**fetchItemsInBagStore**, itemsToResolve) = fetchItems.partition(bagStoreSettings.nonEmpty && _.getUrl.startsWith(bagStoreSettings.get.baseUrl))

If **fetchItemsInBagStore** is empty, we never come to this case:
 `case (itemsFromBagStore, false)`